packages/docs/src/pages/writing-css.mdx

```diff
@@ -6,17 +6,17 @@ order: 4
 
 # Writing CSS
 
-Write CSS using template literals and objects with any of the APIs available with Compiled.
+Write CSS using object styles and objects with any of the APIs available with Compiled.
 
 ```jsx
 import { styled } from '@compiled/react';
 
-const Button = styled.button`
+const Button = styled.button({
   font-size: 10px;
```

Comment:
`font-size: 10px;` should be `fontSize: '10px',`

and so on for the other examples

Reply:
good catch!